packages/babel-core/package.json

```diff
@@ -52,6 +52,7 @@
     "gensync": "^1.0.0-beta.1",
     "json5": "^2.1.2",
     "lodash": "^4.17.13",
+    "mkdirp": "^1.0.4",
```

Comment:
nit: please use `make-dir` to be consistent with `@babel/cli`.

Reply:
Done.